src/components/CardHome.tsx

```diff
@@ -1,23 +1,25 @@
 import React from "react";
 import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
-import styles from "./CardHome.module.css";
 import { CardHomeInterface } from "../interfaces/CardHomeInterface";
 import { Link } from "react-router-dom";
+import styles from "./CardHome.module.css";
 
 const CardHome = (props: CardHomeInterface) => {
   return (
-    <div className={styles.card}>
+    <div className={styles.cardContainer}>
       <Card
-        className="bg-dark text-white"
-        style={{ width: "50vw", height: "50em" }}
+        className={styles.carding}
+        style={{ width: "1000px", height: "800px" }}
```

Comment:
Is it intentional to have a fixed width and height?
I'm not sure what screens your targeting, but this means there's going to be horizontal scroll bars on tablets or even laptops with low resolution.

A more common pattern is something like `width: 90%; max-width: 1100px;` so you're saying "make it 90% of the screen, but don't go above a certain width (as sites tend to look bad when stretched too wide). Or maybe `width: 100%; margin: 0 15px; max-width: 1100px;`.

It's fine if it's intentionally like this, but wanted to make sure.

Reply:
What device(s) is the app supposed to be used for? There's so many hardcoded widths and heights. It's going to be a horrible experience if trying to access this site on a tablet or mobile. Is that okay?